KrisiFy/Entities/ContentEntities/Album.cs

```diff
@@ -10,17 +10,101 @@ namespace KrisiFy.Entities.ContentEntities
     class Album : Playlist
     {
         private Artist artist;
-        private string genre;
-        private DateTime outYear;
-        public Album(string name, DateTime duration, HashSet<Song> songs, Artist artist, string genre, DateTime outYear) : base(name, duration, songs)
+        private List<string> genres;
+        private string outYear;
+        public Album(string name, string duration, List<Song> songs, Artist artist, List<string> genres, string outYear) : base(name, duration, songs)
         {
             this.Artist = artist;
-            this.Genre = genre;
+            this.Genres = genres;
             this.OutYear = outYear;
         }
 
-        public string Genre { get => genre; set => genre = value; }
-        public DateTime OutYear { get => outYear; set => outYear = value; }
-        internal Artist Artist { get => artist; set => artist = value; }
+        public Album(string name) : base(name)
+        {
+
+        }
+
+        public void addSong(string albumName,Song songToAdd)
+        {
+            if (Name.Equals(albumName))
+            {
+                if (Songs.Count == 0)
```

Comment:
isn't  this enough
`                    if (Songs.Contains(songToAdd))
                    {
                        Console.WriteLine("Song is already in this album!");
                    }
                    else
                    {
                        Songs.Add(songToAdd);
                        Console.WriteLine("Song added in album!");
                    }`